### DISCUSSION TITLE
Linux Mint -> Start RustDesk BEFORE Login

### DISCUSSION THREAD

**Discussion by Joaquim3** [rustdesk/rustdesk#7100] [General]
Hi,

-> LinuxMint Virginia  +  RustDesk 123

**I want RUSTDESK to start before the Login screen**.

-> Reason -> I want to be able to reboot the PC remotely and type my login & password.

-> Tests -> I tried "**systemctl enable rustdesk.service**" on the remote computer -> but My remote computer does not AUTOSTART RUSTDESK

-> I'm obliged to login manually on the remote computer, then it starts.

Thanks for your Help,

**Comment by xlionjuan** [comment]
Try use root to ``systemctl enable rustdesk.service`` or use SSH to login to this computer to check the logs.

``systemctl status rustdesk.service``

**Comment by Joaquim3** [comment]
Thanks a lot Xlionjuan.

In fact, my Network Connection was **NOT** started before LOGIN. So I could even NOT SSH my remote computer.

Solution -> in the remote computer I did : 

1. **nmcli con modify 'MYCONNECTIONNAME' connection.permissions ''**
2. **reboot**

Now I can access the remote computer, both by SSH and by RUSTDESK.

Thanks once again, :-)
